pkg/collector/corechecks/orchestrator/pod/pod.go

```diff
@@ -167,6 +168,9 @@ func (c *Check) Run() error {
 	}
 
 	groupID := nextGroupID()
+	metadataAsTags := utils.GetMetadataAsTags(c.cfg)
```

Comment:
Should we use `podName` and `podVersion` constants as in the cluster check?
https://github.com/DataDog/datadog-agent/blob/453fc36fa12026e4738cc64d6fb804fed1cf22df/pkg/collector/corechecks/cluster/orchestrator/collectors/k8s/pod_unassigned.go#L46

This way we don't need to hardcode `"pods"` and avoid future regressions  if `getResourceType` or `GetResourceAnnotations` changes.

Reply:
thank you for the review 
https://github.com/DataDog/datadog-agent/pull/39541/commits/1c92c1a7f06f4deb376098bb1a167d0ec7cdba38

Reply:
I see why you not used it in the first place. The constants defined [here](https://github.com/DataDog/datadog-agent/blob/634cc25dead676d817f0dd00ca2280949b2abda7/pkg/collector/corechecks/cluster/orchestrator/collectors/k8s/types.go#L15) are private to the `cluster` checks package.

I don't think it's a good idea to copy/paste them to the `corechecks` because we'll have a duplicated version that can diverge over time.

For this reason, my suggestion is:
* Let's revert back to use your previous version with hardcoded `"pods"` to unblock this PR
* Let's plan to move the `cluster/orchestrator/collector/k8s/types.go` to a util package that can be used by both cluster and node checks. This can be done in a separate PR.

Wdyt?